### DISCUSSION TITLE
Replace GitHub With CodeCommit

### DISCUSSION THREAD

**Discussion by Sandeepkute** [serverpod/serverpod#563] [Q&A]
@vlidholt @yahu1031 @II11II 
Can we replace GitHub with AWS CodeCommit to deploy our project ??
If possible give the basic steps to do

**Comment by yahu1031** [comment]
There may a chance you can do but I'm not sure how CI/CD works there. You can give it a shot by yourself byt adding existing terraforms to the respective workflow folders in CodeCommit. @Sandeepkute

**Reply by yahu1031** [reply]
But this requirement is a very user(individual) specific decision.
